csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -158,6 +158,53 @@ private HashSet<string> AddFrameworkDlls(HashSet<string> dllPaths)
         {
             var frameworkLocations = new HashSet<string>();
 
+            var frameworkReferences = Environment.GetEnvironmentVariable(EnvironmentVariableNames.DotnetFrameworkReferences);
+            var frameworkReferencesUseSubfolders = Environment.GetEnvironmentVariable(EnvironmentVariableNames.DotnetFrameworkReferencesUseSubfolders);
+            _ = bool.TryParse(frameworkReferencesUseSubfolders, out var useSubfolders);
+            if (!string.IsNullOrWhiteSpace(frameworkReferences))
+            {
+                RemoveFrameworkNugetPackages(dllPaths);
+                RemoveNugetPackageReference(FrameworkPackageNames.AspNetCoreFramework, dllPaths);
+                RemoveNugetPackageReference(FrameworkPackageNames.WindowsDesktopFramework, dllPaths);
+
+                var frameworkPaths = frameworkReferences.Split(FileUtils.NewLineCharacters, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Should we split on newline or something else? An option would be using `:` on posix and `;` on windows.

Reply:
I prefer using `:` and `;` - maybe it is possible to use `Path.PathSeparator`.